server/app/controllers/admin/NorthStarProgramCardPreviewController.java

```diff
@@ -0,0 +1,76 @@
+package controllers.admin;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import auth.CiviFormProfile;
+import auth.ProfileUtils;
+import com.google.common.collect.ImmutableList;
+import controllers.CiviFormController;
+import java.util.concurrent.ExecutionException;
+import javax.inject.Inject;
+import org.pac4j.play.java.Secure;
+import play.i18n.Lang;
+import play.i18n.Messages;
+import play.i18n.MessagesApi;
+import play.mvc.Http;
+import play.mvc.Result;
+import repository.VersionRepository;
+import services.applicant.ApplicantPersonalInfo;
+import services.applicant.ApplicantPersonalInfo.Representation;
+import services.applicant.ApplicantService.ApplicantProgramData;
+import services.program.ProgramDefinition;
+import services.program.ProgramService;
+import services.settings.SettingsManifest;
+import views.admin.programs.NorthStarProgramCardPreview;
+
+/** Controller for rendering a program card preview. */
+public final class NorthStarProgramCardPreviewController extends CiviFormController {
+  private final NorthStarProgramCardPreview northStarProgramCardPreview;
+  private final Messages messages;
+  private final SettingsManifest settingsManifest;
+  private final ProgramService programService;
+
+  @Inject
+  public NorthStarProgramCardPreviewController(
+      ProfileUtils profileUtils,
+      VersionRepository versionRepository,
+      NorthStarProgramCardPreview northStarProgramCardPreview,
+      MessagesApi messagesApi,
+      SettingsManifest settingsManifest,
+      ProgramService programService) {
+    super(profileUtils, versionRepository);
+    this.northStarProgramCardPreview = checkNotNull(northStarProgramCardPreview);
+    this.messages = messagesApi.preferred(ImmutableList.of(Lang.defaultLang()));
+    this.settingsManifest = checkNotNull(settingsManifest);
+    this.programService = checkNotNull(programService);
+  }
+
+  @Secure
```

Comment:
```suggestion
  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
```
(question, blocking) Will this be used by other things? Only CiviForm admins are able to create programs. So it's that's the only place it'll be used, up the authorizers.